packages/next-tinacms-azure/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "next-tinacms-azure",
+  "version": "0.1.0",
+  "description": "",
+  "main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"files": [
+		"dist"
+	],
+	"typings": "dist/index.d.ts",
+	"license": "Apache-2.0",
+	"buildConfig": {
+		"entryPoints": [
+			"src/index.ts",
+			{
+				"name": "src/handlers.ts",
+				"target": "node"
+			}
+		]
```

```suggestion
		"entryPoints": [
			"src/index.ts",
			{
				"name": "src/handlers.ts",
				"target": "node"
			},
			{
                "name": "src/delivery-handlers.ts",
                "target": "node"
			}
		]
```
export the `delivery-handlers`